standard/info/wikis/fighters/info.lua

```diff
@@ -648,6 +648,19 @@ return {
 				lightMode = 'KILL la KILL - IF default allmode.png',
 			},
 		},
+		koihime = {
```

Comment:
On Commons, it appears there is an additional game called [Koihime Enbu RyoRaiRai](https://liquipedia.net/commons/File:Koihime_Enbu_RyoRaiRai_logo.png) that is bieng [used](https://liquipedia.net/fighters/Template:Koihime_Enbu_RyoRaiRai_Tournament) on the Fighters Wiki. 

I do not see this game anywhere on this module.

Reply:
RyoRaiRai is considered as a version of Koihime Enbu. In cases like this, we use the logo of the version in tournament overview pages, but the tournament pages themselves are under the "parent" game. Similar cases are Guilty Gear Xrd (SIGN/REVELATOR/REV2) and Street Fighter IV and V (with their multiple versions)